pygmt/tests/test_datasets_samples.py

```diff
@@ -180,6 +180,7 @@ def test_load_notre_dame_topography():
     assert data["z"].max() == 960
 
 
+@pytest.mark.benchmark
 def test_earth_relief_holes():
     """
     Check that the @earth_relief_20m_holes.grd dataset loads without errors.
```

Comment:
Only benchmarking `test_earth_relief_holes` since it calls both `pygmt.which` and `pygmt.io.load_dataarray`. The other sample datasets call `pygmt.which` and `pd.read_csv` only.